plugins/svg/src/SvgFeatureRenderer/components/SvgFeatureRendering.tsx

```diff
@@ -313,8 +308,13 @@ const SvgFeatureRendering = observer(function SvgFeatureRendering(props: {
   )
 
   useEffect(() => {
-    setHeight(layout.getTotalHeight())
-  }, [layout])
+    // @ts-expect-error
+    const { height } = displayModel
+
+    if (height) {
+      setHeight(height)
+    }
+  }, [displayModel])
```

Comment:
this technically has a useeffect dependency on displayModel.height. that might be enough to fix the race condition. i still think the "proper" thing i feel like is for it to be the block height but it might be workable with just the display height here